theforeman.org/yaml/builders/smart-proxy-plugin.yaml

```diff
@@ -13,7 +13,12 @@
           rvm gemset empty --force
           set -x
 
-          gem install bundler --no-document
+          if ["${{ruby}}" = '2.7']
```

Comment:
Annoyingly, with the quoted string in bash, we needed to manually do the double bracketing instead of jenkins job builder doing it for us in the .yaml configs.  This was the cause of previous job failures.